system/include/compat/emmintrin.h

```diff
@@ -433,11 +433,11 @@ _mm_cvttpd_epi32(__m128d __a)
   int m[2];
   for(int i = 0; i < 2; ++i)
   {
-    float elem = __a[i];
-    if ((lrint(elem) != 0 || fabs(elem) < 2.0) && !isnanf(elem) && elem <= INT_MAX && elem >= INT_MIN)
+    double elem = __a[i];
+    if ((lrint(elem) != 0 || fabs(elem) < 2.0) && !isnanf(elem) && elem < 2147483648.0 && elem >= -2147483648.0)
```

Comment:
These hardcoded constants seem worse than using predefined macros.    Are there no macros for these?